e2etests/features/eyfs_p1_communication_and_language.feature

```diff
@@ -11,8 +11,8 @@ Feature: EYFS - Communication and Language - Links
     Then page has heading "Communication and language"
     Then "Left menu pane" item "1" is "Communication and language"
     And the following "links" are available:
-      | Interactions[3 times]                |
```

Comment:
This should not have been changed as the vertical bars no longer line up.  The purpose of this is it's documentation, and more readable when aligned.

Reply:
Done !